conductor-client/src/main/kotlin/com/openlattice/postgres/external/ExternalDatabasePermissioner.kt

```diff
@@ -258,7 +260,30 @@ class ExternalDatabasePermissioner(
             columnAcls: List<Acl>,
             columnsById: Map<AclKey, TableColumn>
     ) {
-        updateTablePermissions(action, columnAcls, columnsById, TableType.VIEW)
+        // add acl/TableColumn associated to EdmConstants.ID_FQN (with uuid IdConstants.ID_ID.id)
+        val completedColumnAcls = columnAcls.toMutableList()
+        val completedColumnsById = columnsById.toMutableMap()
+        columnAcls.forEach { 
+            val idFqnAclKey = AclKey(it.aclKey[0], IdConstants.ID_ID.id)
+            val org_id = columnsById.getValue(it.aclKey).organizationId
```

Comment:
Our style conventions are to use camelCase.

```suggestion
            val orgId = columnsById.getValue(it.aclKey).organizationId
```